.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ '**' ]
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
you can't use $default-branch here, this macro is only for GH Action templates (and will be replaced with the actual default branch of the repo) so replace with `main`

Reply:
Updated. Probably main was used by default, so action passed successfully.